src/package/packageVersionCreate.ts

```diff
@@ -419,6 +421,12 @@ export class PackageVersionCreate {
     return this.createRequestObject(preserveFiles, packageVersTmpRoot, packageVersBlobZipFile);
   }
 
+  private verifyHasSource(componentSet: ConvertResult): void {
```

Comment:
Verify that the component set has source present
Wrote a function so it was simple to mock